### DISCUSSION TITLE
Laravel Installation Fail

### DISCUSSION THREAD

**Discussion by nyan-ucsp** [laravel/framework#34704] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/59817342/95216989-b727ac80-0818-11eb-944c-2d7baf6a0211.PNG)
Please help me what should I do.

**Comment by mfn** [comment]
Can't read that colors. Please don't paste screenshots of terminal output, use tripple-backticks and enclose the output there, thanks!

**Comment by GrahamCampbell** [comment]
It looks like you have a file permissions issue, no disk space left, or antivirus software messing things up.

**Reply by nyan-ucsp** [reply]
yes, windows auto delete the downloaded zip file. Thanks.

**Comment by nyan-ucsp** [comment, marked as answer]
This error occurs because window real-time protection , window auto delete symfony/console zip file after downloaded. So I closed real-time protection and installed Laravel successfully. Thanks for your discussions.
